### ISSUE TITLE
Docs: Authenticate Mediation vs Conditional Param

### ISSUE THREAD

**Issue by hjaber** [passwordless-id/webauthn#64]
[docs/authentication.md](https://github.com/passwordless-id/webauthn/blob/8fdc63af65d527cb5cb3618271cd52f2641a2f77/docs/authentication.md?plain=1#L69) `client.authenticate()`parameter should be `'conditional'` instead of `'mediation'`
```md
| `mediation` | | See https://developer.mozilla.org/en-US/docs/Web/API/CredentialsContainer/get#mediation
```

[docs/faq.md](https://github.com/passwordless-id/webauthn/blob/8fdc63af65d527cb5cb3618271cd52f2641a2f77/docs/faq.md?plain=1#L97)
```md
...with `mediation: true` as option.
```

```ts
interface AuthenticateOptions extends CommonOptions {
    allowCredentials?: CredentialDescriptor[];
    conditional?: boolean;
}
```

[src/client.ts](https://github.com/passwordless-id/webauthn/blob/8fdc63af65d527cb5cb3618271cd52f2641a2f77/src/client.ts#L178)
```ts
const raw = await navigator.credentials.get({
        publicKey: authOptions,
        mediation: options.conditional ? 'conditional' : undefined,
        signal: ongoingAuth?.signal
    }) as PublicKeyCredential
```

Would it be easier to follow the webauthn [API](https://developer.mozilla.org/en-US/docs/Web/API/CredentialsContainer/get#mediation) of using `mediation` with a default value of `'optional'` and allowing `'conditional' | 'optional' | 'required' | 'silent'`?

I don't have a strong opinion on the param name but as I'm working through migrating to V2, it was `'mediation'` in V1 but for some reason I'm having trouble getting autocomplete to work in V2.

**Comment by dagnelies** [comment]
I hesitated regarding this one too. I'll fix the inconsistencies. By the way, there is a demo on the front page to do just that. Have you seen it?

**Comment by hjaber** [comment]
Oh yeah the demo worked great. I discovered the issue that was preventing my autocomplete, a password manager. They make passkeys more complicated.

**Comment by dagnelies** [comment]
Well, I changed it completely into `autocomplete:true`. The docs are already updated but the package is not yet released.

**Comment by dagnelies** [comment]
Nevermind... I still have a detail to fix.

**Comment by dagnelies** [comment]
should be fixed in 2.0.2

**Comment by hjaber** [comment]
Fixed in 2.0.2 with API change to autocomplete:true
